askama_parser/src/tests.rs

```diff
@@ -918,3 +918,17 @@ fn fuzzed_unary_recursion() {
     const TEMPLATE: &str = include_str!("../tests/unary-recursion.txt");
     assert!(Ast::from_str(TEMPLATE, None, &Syntax::default()).is_err());
 }
+
+#[test]
```

Comment:
Is there no annotation for `#[test]` to limit the runtime?

Reply:
No: https://doc.rust-lang.org/reference/attributes/testing.html.